scripts/split_n_preprocess.py

```diff
@@ -6,90 +6,44 @@
 import os
 import numpy as np
 import pandas as pd
-import pickle
-from sklearn.model_selection import train_test_split
 from sklearn import set_config
-from sklearn.model_selection import train_test_split
-from sklearn.preprocessing import StandardScaler
-from sklearn.compose import make_column_transformer, make_column_selector
-
+sys.path.append(os.path.join(os.path.dirname(__file__), '..'))
+from src.split_train_test import split_train_test_data, validate_split_data
+from src.clean_data import write_data
+from src.preprocessor import create_save_preprocessor
 
 @click.command()
-@click.option('--raw-data', type=str, help="Path to raw data")
+@click.option('--cleaned-data', type=str, help="Path to cleaned data")
+@click.option('--train-data-size', type=str, help="Size of the dataset to include in the train split")
```

Comment:
Can you update the `Makefile` the the necessary changes to run it with this update to the script (that we can not edit the train data size)?